components/com_tags/models/tag.php

```diff
@@ -178,7 +178,7 @@ protected function populateState($ordering = 'c.core_title', $direction = 'ASC')
 		$this->setState('params', $params);
 
 		// Load state from the request.
-		$ids = (array) $app->input->get('id', array());
+		$ids = (array) $app->input->get('id', array(),'int');
```

Comment:
```suggestion
		$ids = (array) $app->input->get('id', array(), 'int');
```
Fix code style error.